boa/src/builtins/symbol/mod.rs

```diff
@@ -100,7 +100,40 @@ impl Symbol {
 
     /// Initialise the `Symbol` object on the global object.
     #[inline]
-    pub fn init(global: &Value) -> (&str, Value) {
+    pub fn init(interpreter: &mut Interpreter) -> (&'static str, Value) {
+        let symbol_async_iterator = Symbol(
+            Some("Symbol.asyncIterator".into()),
+            interpreter.generate_hash(),
+        );
+        let symbol_has_instance = Symbol(
+            Some("Symbol.hasInstance".into()),
+            interpreter.generate_hash(),
+        );
+        let symbol_is_concat_spreadable = Symbol(
+            Some("Symbol.isConcatSpreadable".into()),
+            interpreter.generate_hash(),
+        );
+        let symbol_iterator = Symbol(Some("Symbol.iterator".into()), interpreter.generate_hash());
+        let symbol_match = Symbol(Some("Symbol.match".into()), interpreter.generate_hash());
+        let symbol_match_all = Symbol(Some("Symbol.matchAll".into()), interpreter.generate_hash());
```

Comment:
I can't see the documentation for this in the spec?

Reply:
They're listed here https://www.ecma-international.org/ecma-262/10.0/index.html#sec-well-known-symbols. I can add a comment later

Reply:
I meant specifically for matchAll?

Reply:
Ah, you're right in that it isn't in the ecma-international spec, however it is in the tc39 one (https://tc39.es/ecma262/#sec-well-known-symbols) which I must have referenced when implementing it. I'm not sure which one I should be going off so I can update it if needed

Reply:
Ah right it must be new in the 2021 spec - keep it and if possible could you please update the link to the new spec 😃